Makefile

```diff
@@ -191,7 +191,7 @@ manually_upload_certs: check-tf-version ## Manually upload to AWS the SSL certif
 pingdom: check-tf-version ## Use custom Terraform provider to set up Pingdom check
 	$(if ${ACTION},,$(error Must pass ACTION=<plan|apply|...>))
 	$(eval export PASSWORD_STORE_DIR?=~/.paas-pass)
-	$(eval export PINGDOM_CONTACT_IDS=11089310)
+	$(eval export PINGDOM_CONTACT_IDS=11089310,11189971)
```

Comment:
11189971 is the id for 24x7 notification
I've also created 11190300 for in hours notifications
Could you add logic to have different ids for different environments?